hicommonwealth/commonwealth
# Disable Editor with tooltip

## Link to Issue
Closes: #4377 


## Description of Changes
- because we don't have new tooltip yet, I just used the old tooltip and styled it to match the design
- added `not-allowed` cursors for submit button and editor

https://github.com/hicommonwealth/commonwealth/assets/14819225/9a646bfc-9f18-4bcf-a959-11f48e9f4ca9


## Test Plan
- go to community that you are not part of 
- check cursor on disabled editor and button
- tooltip should be visible over the disabled editor

## Other Considerations

- #4380 should be merged first!
